src/main/kotlin/uk/gov/justice/digital/hmpps/managerecallsapi/config/ManageRecallsApiExceptionHandler.kt

```diff
@@ -70,24 +70,37 @@ class ManageRecallsApiExceptionHandler {
         .body(ErrorResponse(NOT_FOUND, e.toString()))
     }
 
+  @ExceptionHandler(ClientException::class)
+  fun handleException(e: ClientException): ResponseEntity<ErrorResponse> =
```

Comment:
No addition to `ManageRecallsApiExceptionHandlerTest`?  Looking at that there seems scope to add tests.